src/vivarium_uw_covid/components/covid_deaths.py

```diff
@@ -0,0 +1,110 @@
+"""Methods for modeling COVID mortality,
+calibrated to IHME COVID Projections
+
+Author: Abraham D. Flaxman
+Date: 2020-07-08
+"""
+
+import numpy as np, pandas as pd
+import scipy.interpolate, pymc as pm
+
+
+# TODO: refactor this into loader.py
+dir_name = '/home/j/Project/simulation_science/covid/data'
+df_fac_staff = pd.read_csv(f'{dir_name}/uw_staff_age_sex_counts.csv')
+df_fac_staff['p'] = df_fac_staff.value / df_fac_staff.value.sum()
+
+rate_dir = '2020_06_29.01'
+df = pd.read_csv(f'/ihme/covid-19/rates/{rate_dir}/ifr_preds_1yr.csv')
+f_ifr_ = scipy.interpolate.interp1d(df.age_mid.values, pm.invlogit(df.lowest_ifr), kind='linear', fill_value='extrapolate')
```

Comment:
Doing computation/io at import time is bad practice.  Say, e.g., the file system was broken for some reason.  Importing anything from this module (or anything that imports anything from this module) would hang or break in a really unhelpful way.

Reply:
agree!  I would like to refactor this in to the loader, and create an artifact (or netCDF) to be robust against this.